packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Tabs/configAggregator.js

```diff
@@ -0,0 +1,40 @@
+import {
+    getAdvanced,
+    getBackgroundImages,
+    getCssClasses,
+    getVerticalAlignment
+} from '../../utils';
+
+export default node => {
+    const navigationEl = node.childNodes[0];
+    const navigation = {
+        style: {
+            ...getAdvanced(navigationEl)
+        },
+        ...getCssClasses(navigationEl)
+    };
+
+    const headerEls = navigationEl.childNodes;
+    const headers = [].map.call(headerEls, headerEl => headerEl.textContent);
```

Comment:
Slightly more self-describing

```js
const headers = Array.from(headerEls, headerEl => headerEl.textContent);
```

Reply:
This will be resolved as part of the Tabs story.